changelog.d/20240103_085440_nicolas.henin_runtimeStatus.md

```diff
@@ -0,0 +1,3 @@
+### @marlowe.io/runtime-rest-client
+
+- Added a new endpoint `GetRuntimeStatus` that retrieves informations about the Runtime (version deployed, Network Id of the Node and tips)
```

Comment:
Please add this as part of healthcheck response instead of creating a new endpoint